lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -119,6 +119,27 @@ def log_fail_fast_limit_met
 
         def post_trap_signals
           RSpec.world.wants_to_quit = true
+
+          print_current_rspec_batch_command
+        end
+
+        def post_log_threads(threads)
+          threads.each do |thread|
+            next unless thread.backtrace
+
+            spec_file_lines = thread.backtrace.select { |line| line.include?('_spec.rb') }
+            next if spec_file_lines.empty?
```

Comment:
suggestion

`empty?` feels to broad, what about `.nil?` or `unless spec_file_lines`?

Reply:
`select` returns `[]` so empty? is fine.